modules/openideal_toolbar/openideal_toolbar.links.menu.yml

```diff
@@ -0,0 +1,52 @@
+openideal_toolbar.help:
+  title: 'OpenideaL'
+  route_name: openideal_toolbar.system
+  menu_name: admin
+  parent: system.admin
+  weight: 4
+
+openideal_toolbar.all_challenges_page:
```

Comment:
View configuration allows us to provide menu links directly via View settings. These links will be translatable and more reliable.